### ISSUE TITLE
<select> tag's 'required' not remove

### ISSUE THREAD

**Issue by Nagomaro** [ncri/nested_form_fields#94]
An error when remove. When 'required' is inside <select> tag, 'required' not remove

**Comment by ncri** [comment]
Sorry, I don't understand, can you give me an example?

**Comment by Nagomaro** [comment]
Sorry for my poor English.

Example code (Rails5.0) :
```
= f.nested_fields_for :items, wrapper_tag: :tr do |ff|
    td = ff.select :category_id, options_for_select(@categories.map{ |t| [t.category_name, t.id]}), {}, {class: 'form-control category_id', required: 'true'}
    td = ff.number_field :quantity, class: 'form-control quantity', max: 999999, min: 0, required: 'true'
    td = ff.number_field :unit_cost, class: 'form-control unit_cost', max: 99, min: 0, required: 'true'
    td.text-center = ff.remove_nested_fields_link 'Remove', class: 'btn btn-danger', role: 'button'
```

When 'Remove' button is pressed, 'required' of `<input>` tag remove, but 'required' of `<select>` tag does not remove.
so an error will occur when committing.

**Comment by ncri** [comment]
Ah, I understand now, thanks. I released a new version 0.8.2, can you test that? Does it work there?

**Comment by Nagomaro** [comment]
I tested with version 0.8.2, the problem is resolved.

Thank you for your modify!
